HEP/src/cli.ts

```diff
@@ -37,52 +32,54 @@ async function getDataSetPayload(
     dataSetId: string
 ): Promise<MetadataPayload> {
```

Comment:
This scripts is now pretty simple and it's ok to have this `getDataSetPayload`. Whenever it grows a bit more, I'd move all this logic somewhere else and leave cli.js as simple as possible, just the processing of arguments and single calls to methods that do all the work.